diff --git a/node_modules/react-native-css-interop/.cache/android.js b/node_modules/react-native-css-interop/.cache/android.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/ios.js b/node_modules/react-native-css-interop/.cache/ios.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/macos.js b/node_modules/react-native-css-interop/.cache/macos.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/native.js b/node_modules/react-native-css-interop/.cache/native.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/windows.js b/node_modules/react-native-css-interop/.cache/windows.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/dist/runtime/native/render-component.js b/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
index eba25ec..68c8f5b 100644
--- a/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
+++ b/node_modules/react-native-css-interop/dist/runtime/native/render-component.js
@@ -122,21 +122,29 @@ function printUpgradeWarning(warning, originalProps) {
 }
 function stringify(object) {
     const seen = new WeakSet();
-    return JSON.stringify(object, function replace(_, value) {
-        if (!(value !== null && typeof value === "object")) {
-            return value;
-        }
-        if (seen.has(value)) {
-            return "[Circular]";
-        }
-        seen.add(value);
-        const newValue = Array.isArray(value) ? [] : {};
-        for (const entry of Object.entries(value)) {
-            newValue[entry[0]] = replace(entry[0], entry[1]);
-        }
-        seen.delete(value);
-        return newValue;
-    }, 2);
+    try {
+        return JSON.stringify(object, function replace(_, value) {
+            if (!(value !== null && typeof value === "object")) {
+                return value;
+            }
+            if (seen.has(value)) {
+                return "[Circular]";
+            }
+            seen.add(value);
+            const newValue = Array.isArray(value) ? [] : {};
+            try {
+                for (const entry of Object.entries(value)) {
+                    newValue[entry[0]] = replace(entry[0], entry[1]);
+                }
+            } catch (_) {
+                return "[unserializable]";
+            }
+            seen.delete(value);
+            return newValue;
+        }, 2);
+    } catch (_) {
+        return "[unserializable]";
+    }
 }
 function getDebugReplacer() {
     const seen = new WeakSet();
